devtools/gradle/settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = projectName
+rootProject.name = quarkus-gradle-plugin
```

Comment:
if we change this we should remove gradle.properties as well since it wont be needed any longer

Reply:
This was actually already done in #1405 (as `'quarkus-gradle-plugin'`, with single quotes), so I'll just rebase this PR on master. I've also removed the `gradle.properties` as suggested. Hope this one is good to merge now!